website/content/usage/example.md

```diff
@@ -20,32 +20,71 @@ address pools, and set BGP attributes to control the visibility of
 each set of addresses:
 
 ```yaml
-# Rest of config omitted for brevity
-bgp-communities:
-  # Our datacenter routers understand a "VPN only" BGP community.
-  # Announcements tagged with this community will only be propagated
-  # through the corporate VPN tunnel back to developer offices.
-  vpn-only: 1234:1
-address-pools:
-- # Production services will go here. Public IPs are expensive, so we leased
-  # just 4 of them.
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
   name: production
-  protocol: bgp
+  namespace: metallb-system
+spec:
+  # Production services will go here. Public IPs are expensive, so we leased
+  # just 4 of them.
   addresses:
   - 42.176.25.64/30
+```
 
-- # On the other hand, the sandbox environment uses private IP space,
-  # which is free and plentiful. We give this address pool a ton of IPs,
-  # so that developers can spin up as many sandboxes as they need.
+```yaml
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
   name: sandbox
-  protocol: bgp
+  namespace: metallb-system
+spec:
   addresses:
+  # On the other hand, the sandbox environment uses private IP space,
+  # which is free and plentiful. We give this address pool a ton of IPs,
+  # so that developers can spin up as many sandboxes as they need.
   - 192.168.144.0/20
-  bgp-advertisements:
-  - communities:
+```
+
+Then we advertise them, and set BGP attributes to control the visibility of
+each set of addresses:
+
+```yaml
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
+metadata:
+  name: local
+spec:
+  ipAddressPools:
+  - production
+```
+
+```yaml
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
+metadata:
+  name: external
+spec:
+  ipAddressPools:
+  - sandbox
+  communities:
     - vpn-only
 ```
 
+```yaml
+# Our datacenter routers understand a "VPN only" BGP community.
+# Announcements tagged with this community will only be propagated
+# through the corporate VPN tunnel back to developer offices.
+apiVersion: metallb.io/v1beta1
+kind: Community
+metadata:
+  name: vpn-only
```

Comment:
regarding the Community metadata.name, IMO having it as "vpn-only", the same as the community alias name
can confuse the user and we should call it something else so it will be clearer that the Community alias in the bgpAdv is pointing to the one in the Spec, and not in the Metadata.